### PR TITLE
Features/url base localization

### PR DESCRIPTION
**Pull request by plbonneville**

### CODE REVIEW

**Review by pierluc-ssoucy — CHANGES_REQUESTED** [review]
Two other similar item I was not sure where to put, and not sure if it's caused by localized routes or by authorization redirects but for both "approval pages" it seems to get stuck on the root splash page after you accept/finish the below:
- [ ] Health and safety measures
- [ ] Privacy Act Statement

File: src/WebApp/Shared/NavMenu.razor [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
                 </span>
-                @Localizer["Home"]
+                @Localizer["LanguageToggle"]
```
[the remark this answers is not in the document]
Comment (plbonneville) [reply src/WebApp/Shared/NavMenu.razor:18]: Agreed.

File: src/WebApp/Pages/Index.razor.cs [hunk 2]
```diff
@@ -15,99 +15,34 @@ namespace OfficeEntry.WebApp.Pages
     public partial class Index : ComponentBase
     {
         [Inject] public NavigationManager NavigationManager { get; set; }
-        [Inject] public IMediator Mediator { get; set; }
-        [Inject] public ILocalStorageService LocalStorage { get; set; }
-        [Inject] public IStringLocalizer<App> Localizer { get; set; } 
 
-        protected override async Task OnAfterRenderAsync(bool firstRender)
+        protected override void OnInitialized()
```
[the remark this answers is not in the document]
Comment (plbonneville) [reply src/WebApp/Pages/Index.razor.cs:19]: They have been removed.

**Comment by pierluc-ssoucy** [comment]
@plbonneville we you able to fix the "redirect bug" mentioned in the global comment?

**Pull request merged**